test/browser.js

```diff
@@ -171,27 +172,48 @@ test('throws if does not support ReadableStream', withPage, async (t, page) => {
 	await server.close();
 });
 
-test('FormData with searchParams', withPage, async (t, page) => {
-	t.plan(2);
-
+test.failing('FormData with searchParams', withPage, async (t, page) => {
```

Comment:
I think it would be good to leave the existing test in place and add this as a _new_ test. That should help illustrate the specifics of what is broken.